recipe/meta.yaml

```diff
@@ -1,62 +1,95 @@
 # version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
 #     This version number follows the "platform toolset"
-{% set vcver="14" %}
-{% set vsyear="2015" %}
-{% set fullver="14.0.25420" %}
+{% set vcver = "14.1" %}
+{% set vsyear = "2017" %}
+{% set fullver = "14.16.27012" %}
+# VS2017 is fundamentally compatible with VS2015.  We name our package vs2015_runtime so that it can't be
+#    mixed up with the runtime from VS2015 - you do actually need VS2017 runtime for things compiled with
+#    VS2017, and the runtime we have here is backwards-compatible with things compiled with VS2015.
+{% set runtime_year = "2015" %}
 
 package:
   name: vs{{ vsyear }}
   version: {{ fullver }}
 
 build:
-  number: 11
-  skip: True  [not win]
+  number: 0
+  skip: True  # [not win]
 
 outputs:
   - name: vs{{ vsyear }}_{{ cross_compiler_target_platform }}
+    version: {{ cl_version }}
     script: install_activate.bat
-    track_features:
-      - vc{{vcver}}
-    run_exports:
-      strong:
-        # compatible within major version.  This is MSFT's incrementing of the UCRT version.  It was
-        #    14.0 with VS2015, and 14.1 with VS2017 - if the major version changes, we're assuming
-        #    that binary compatibility breaks.
-        - {{ pin_subpackage('vc') }}
-        - {{ pin_subpackage('vs' ~ vsyear ~ '_runtime', max_pin=None) }}
+    build:
+      track_features:
+        - vc{{ vcver.split(".")[0] }}
+      run_exports:
+        strong:
+          # compatible within major version.  This is MSFT's incrementing of the UCRT version.  It was
+          #    14.0 with VS2015, and 14.1 with VS2017 - if the major version changes, we're assuming
+          #    that binary compatibility breaks.  This has a minimum bound equal to vcver, so building with
+          #    vs2017 here will require the vs2017-era runtime (the package is named vs2015_runtime, but the
+          #    version differs)
+          - {{ pin_subpackage('vc') }}
+          - {{ pin_subpackage('vs' + runtime_year + '_runtime') }}
```

Comment:
This used to be `max_pin=None`. Should it go back?

Reply:
bump on this run exports comment too

Reply:
Yes. I guess they'll break compatibility in 15